client/stylesheets/_toCat.scss

```diff
@@ -5189,6 +5264,7 @@ On click the images will be displayed at normal size to complete the effect
 
 .headerSearchRadiusLogo a {
 	font-size: 1.3rem !important;
+	color: #404040 !important;
```

Comment:
Color literals like `#404040` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented with spaces, not tabs